core/ir/aarch64/instr_create_api.h

```diff
@@ -1759,6 +1759,19 @@ enum {
 #define INSTR_CREATE_fcvtzu_vector(dc, Rd, Rm, width) \
     instr_create_1dst_2src(dc, OP_fcvtzu, Rd, Rm, width)
 
+/**
+ * Creates a FCVTZU vector floating-point to fixed-point convert instruction.
```

Comment:
Same comment as on Matt's PR: s/a/an/ for most common pronunciation.  Ditto below.